src/rust/core/server/src/threads/workers/multi.rs

```diff
@@ -186,7 +207,7 @@ where
                 let mut message = TokenWrapper::new(request, token);
 
                 for retry in 0..QUEUE_RETRIES {
-                    if let Err(SendError::Full(m)) = self.storage_queue.try_send(message) {
+                    if let Err(m) = self.storage_queue.try_send_to(0, message) {
```

Comment:
Would  it help readers if storage thread id is declared using a constant instead of `0` explicitly?

Reply:
Done